storm-core/src/jvm/org/apache/storm/messaging/netty/StormClientHandler.java

```diff
@@ -60,7 +60,6 @@ public void messageReceived(ChannelHandlerContext ctx, MessageEvent event) {
                 //This should be the metrics, and there should only be one of them
                 List<TaskMessage> list = (List<TaskMessage>)message;
                 if (list.size() < 1) throw new RuntimeException("Didn't see enough load metrics ("+client.getDstAddress()+") "+list);
-                if (list.size() != 1) LOG.warn("Messages are not being delivered fast enough, got "+list.size()+" metrics messages at once("+client.getDstAddress()+")");
```

Comment:
I'm also OK for this approach. If we pick this, comment doesn't need to be modified.